vividus-tests/src/main/resources/story/integration/ElementStepsTests.story

```diff
@@ -39,7 +39,7 @@ Then text `Mouse enter count: 1` exists
 Scenario: Step verification When I click on an element '$searchAttributes' then the page does not refresh
 Given I am on page with URL `${vividus-test-site-url}/mouseEvents.html`
 When I click on an element 'xpath((.//*[@type='radio'])[1])' then the page does not refresh
-Then the page has the relative URL '/mouseEvents.html'
```

Comment:
Should we keep this deprecated step or completely replace it with the new one?

Reply:
while we have the deprecated step in codebase, we should test it